src/hooks/useInitializeAccountData.ts

```diff
@@ -17,11 +16,6 @@ export default function useInitializeAccountData() {
         dispatch(explorerInit());
       });
 
-      InteractionManager.runAfterInteractions(async () => {
-        logger.sentry('Initialize uniqueTokens');
-        await dispatch(uniqueTokensRefreshState());
```

Comment:
no longer need to prefetch nfts bc unlike redux, react query will fetch the nfts from the interweb (or cache if available) on demand